op-chain-ops/genesis/db_migration.go

```diff
@@ -54,8 +54,21 @@ func MigrateDB(ldb ethdb.Database, config *DeployConfig, l1Block *types.Block, m
 		}, nil
 	}
 
-	if config.L2GenesisBlockBaseFeePerGas == nil {
-		return nil, errors.New("must configure L2 genesis block base fee per gas")
+	// Ensure EIP 1559 params are sane
+	if config.EIP1559Elasticity == 0 {
```

Comment:
Isn't there a `Check()` method for the deploy config? If so, should all these checks go in there?

Reply:
I'll consolidate things there, i was worried it would break other things